### PR TITLE
SAK-49519 Portal a11y address insufficient color contrast issues with text or links on Gateway page

### PR DESCRIPTION
**Pull request by kunaljaykam**
https://sakaiproject.atlassian.net/browse/SAK-49519

### CODE REVIEW

File: library/src/webapp/content/gateway/about.html [hunk 1]
```diff
@@ -16,6 +16,10 @@
         height: 150px;
         background-color:#dee2e6;
     }
+    
+    a {
+    color: #0e4466;
```
Comment (bgarciaentornos) [inline library/src/webapp/content/gateway/about.html:21]: Fix spacing

File: library/src/webapp/content/gateway/acknowledgements.html [hunk 2]
```diff
@@ -12,6 +12,12 @@
 	includeWebjarLibrary("bootstrap");
 </script>
 
+<style>
+	a {
+    color: #0e4466;
+	}
```
Comment (bgarciaentornos) [inline library/src/webapp/content/gateway/acknowledgements.html:18]: Fix spacing

File: library/src/webapp/content/gateway/features.html [hunk 3]
```diff
@@ -10,6 +10,12 @@
 	includeWebjarLibrary("bootstrap");
 </script>
 
+<style>
+	.btn-primary {
+    	background-color: #0e4466;
```
Comment (bgarciaentornos) [inline library/src/webapp/content/gateway/features.html:15]: Fix spacing

**Review by bgarciaentornos — CHANGES_REQUESTED** [review]
Thanks, just a couple spacing issues

**Comment by kunaljaykam** [comment]
> Thanks, just a couple spacing issues

Thanks for noticing.

**Review by bgarciaentornos — APPROVED** [review]
Thanks for addressing the issues!

**Pull request merged**